src/enums/viewState.ts

```diff
@@ -88,6 +88,14 @@ export const DismissedViewState = makeEnum({
 export type DismissedViewState =
   typeof DismissedViewState[keyof typeof DismissedViewState];
 
+/**
+ * Consent Manager view state for the Interactive Advertising Bureau
+ */
+export const IABViewState = makeEnum({
+  /* Transparency and Consent Framework for Europe and Canada */
+  TCF: 'TCF',
```

Comment:
maybe in the comment leave Canada out of it since we don't support it? tcf canada is fairly similar to europe but comes with its own set of purposes, vendors, etc

Reply:
i think the UIs are different? there's a lot of vocab differences that should be reflected in the UI but i haven't looked too deeply into the UI

Reply:
Okay, in that case should we have two IAB View States? TCF_EU and TCF_CA? @anotherminh @linh-transcend

Reply:
I don't know. are there plans to support canada in the future? maybe the europe one could just be referred to as TCF_EU to be more specific if we happen to support canada later on?

Reply:
good point linh.

Let's only add `TCF_EU` here for now, which will map to both `tcfeuv1` and `tcfeuv2` in the GPP section names

Reply:
@anotherminh will we continue to support `tcfeuv1`? I was thinking of having `TCF_EU` just map to `tcfeuv2` with the expectation that we could either update that in place or distinguish them later. But v1 and v2 are different, right? Having TCF_EU map to both v1 and v2 seems ambiguous. I'd rather just have it be TCF_EU if we aren't going to distinguish